pkg/simple/client/openpitrix/helmwrapper/helm_wrapper.go

```diff
@@ -258,60 +192,45 @@ func SetMock(mock bool) Option {
 	}
 }
 
+func NewInClusterRESTClientGetter(kubeconfig string, namespace string) genericclioptions.RESTClientGetter {
+	flags := genericclioptions.NewConfigFlags(true)
+	if kubeconfig != "" {
+		cfg, _ := clientcmd.RESTConfigFromKubeConfig([]byte(kubeconfig))
+		flags = genericclioptions.NewConfigFlags(false)
+		flags.APIServer = &cfg.Host
+		flags.BearerToken = &cfg.BearerToken
+		flags.CAFile = &cfg.CAFile
+		flags.WithDiscoveryBurst(cfg.Burst)
+		if sa := cfg.Impersonate.UserName; sa != "" {
+			flags.Impersonate = &sa
+		}
+	}
```

Comment:
The following code will cause a cert error if the kubefonfig includes certificate-authority-data.

```
E0513 11:24:17.485511       1 helm_wrapper.go:107] namespace: z1, name: redis-emjr1v, run command failed, error: Kubernetes cluster unreachable: Get "https://10.233.0.1:443/version?timeout=32s": x509: certificate signed by unknown authority
```

Reply:
Thanks for your reply.in this case,the multi cluster kubeconfig havn't check err,i will fix this err.In addition how can i fast to mock a multi cluster env

Reply:
> Thanks for your reply.the multi cluster kubeconfig havn't check err,i will fix this err.In addition how can i fast to mock a multi cluster env

`sigs.k8s.io/controller-runtime/pkg/envtest` may be helpful for you.

Reply:
Thanks to your help, I created the MemoryRESTClientGetter to solve this problem